### DISCUSSION TITLE
Milo Certification

### DISCUSSION THREAD

**Discussion by ArnoLWeiss** [eclipse-milo/milo#831] [General]
Hi everyone,

when looking up what Profiles the Eclipse Milo stack confirms to, I was not able to find any certification on the [OPC Foundation Homepage](https://opcfoundation.org/certified-products). Am I missing something here? Is certification planned? If no, what profiles COULD be in the scope of the milo project?

Cheers
Arno

**Comment by kevinherron** [comment]
The Milo demo server currently passes the Certification Test Tool suite for the "Standard UA Server" profile and all of the facets that entails.

I may pursue official certification _for the demo server_ when 1.0 is released but there is no certification process for SDKs.

**Comment by GoetzGoerisch** [comment]
@ArnoLWeiss interesting question. As @kevinherron pointed out, SDKs can not be certified, only concrete implementation.

So milo is supposed to fulfil all of those mandatory Profiles/Facets: http://opcfoundation.org/UA-Profile/Server/StandardUA
